azure-storage/spring-cloud-starter-stream-sink-wasb/README.md

```diff
@@ -0,0 +1,17 @@
+# spring-cloud-stream-wasb-sink
+#### A Windows Azure Storage Blob sink module for Spring Cloud Stream
+
+This module is an MVP implementation of a BlockBlob-based sink module
+for text payloads originating from a Spring Cloud Dataflow stream.
+
```

Comment:
Can include auto-generated documentation of properties (see other READMEs in this repo)